Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Rifles/colony_rifles.yml

```
@@ -0,0 +1,52 @@
+- type: entity
+  parent: CMBaseWeaponRifle
+  name: AR10 Assault Rifle
+  id: WeaponRifleAR10
+  description: An earlier version of the more widespread M16 rifle. How one of these ended up here is a mystery of its own. Uses 7.62x51mm rounds.
```

Missing one line of the description from 13
```suggestion
  description: An earlier version of the more widespread M16 rifle. Considered to be the father of the 20th century rifle. How one of these ended up here is a mystery of its own. It is chambered in 7.62x51mm.
```